static/mobile/message-en.json

```diff
@@ -1,13 +1,12 @@
 {
   "title": "",
   "text": [
-"Friend,",
-"On this upcoming Shavuot, we will celebrate the beginning of the conversation that is Torah — and at Sefaria, we celebrate the conversation all year long.",
-"Celebrate the gift of Torah by supporting Sefaria’s mission to bring it into the digital age. We’ve set a goal for 500 new Sefaria Sustainers before Shavuot. Will you become a monthly donor right now?",
-"We're grateful for your support."
+"In honor of Shavuot,",
+"we are counting towards the goal of recruiting 500 new Sefaria Sustainers."
+"If you receive value from the library, please consider giving back with a monthly donation today."
   ],
-  "buttonLink": "https://donate.sefaria.org/give/457760/#!/donation/checkout",
+  "buttonLink": "https://donate.sefaria.org/give/487265/#!/donation/checkout?recurring=1&c_src=mobile",
   "buttonText": "Become a Sustainer",
-  "name": "2023-05-04-shavuot",
+  "name": "2023-05-16-shavuot-donate-banner",
```

Comment:
This doesn't match the template name -- you inverted the u&o in Shavuot -- it should be `2023-05-16-shavout-donate-banner`